src/main/java/org/commcare/formplayer/application/FormSubmissionHelper.java

```diff
@@ -268,8 +268,7 @@ private SubmitResponseBean processFormXml(FormSubmissionContext context) throws
         } catch (HttpClientErrorException e) {
             return getErrorResponse(
                     context.getHttpRequest(), Constants.SUBMIT_RESPONSE_ERROR,
-                    String.format("Form submission failed with error response: %s, %s, %s",
-                            e.getMessage(), e.getResponseBodyAsString(), e.getResponseHeaders()),
+                    String.format("Form submission failed with error response: %s", e.getResponseBodyAsString()),
```

Comment:
agree that it will be helpful to have response code here when the reponse body is empty.

Reply:
Cool updated to use `getMessage` which appears to include the status code in testing.